test-cases/longevity/longevity-alternator-ttl-big-dataset.yaml

```diff
@@ -0,0 +1,66 @@
+test_duration: 3000
+# Run ycsb load using max integer value of 2147483647 (2^31-1) for number of records per client.
+# ( The prepare_write_cmd is just a dummy in order to trigger post_prepare_cql_cmds )
+prepare_write_cmd:
+  - >-
+    bin/ycsb load dynamodb -P workloads/workloadc -threads 1 -p recordcount=11
+    -p fieldcount=8 -p fieldlength=64
+    -p insertstart=0 -p insertcount=10  -p table=usertable_no_lwt
+
+stress_cmd:
```

Comment:
can you have a comment somewhere what's big-dataset mean ? is it 1Tb ? 10Mb ? don't force the reader to do the calculations

Reply:
added `YCSB write stress  2000200300 keys (1TB)`